my-app/src/components/UncontrolledForm/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { formData } from "../../types";
+
+export interface UncontrolledFormProps {
+  onSubmit: ({}: formData) => void;
```

Comment:
```suggestion
  onSubmit: (data: formData) => void;
```
и можно было заюзать один интерфейс просто